app/assets/stylesheets/new_design/dossier_messagerie.scss

```diff
@@ -4,9 +4,19 @@
 
 #dossier-messagerie {
   li {
+    display: flex;
+    align-items: flex-start;
     margin-bottom: 2 * $default-padding;
   }
 
+  .right {
```

Comment:
Oui ça me dit quelque chose, mais ça me choque (le mot est fort, mon sentiment est plus faible :) ), un truc du genre `commentaire-issuer-and-body` ou `commentaire-content` me semblerait plus compréhensible